prov/rxm/src/rxm.h

```diff
@@ -170,9 +170,24 @@ enum rxm_proto_state {
 
 extern char *rxm_proto_state_str[];
 
+/* The structures are derived from ofi_ctrl_hdr and ofi_op_hdr */
+struct rxm_ctrl_hdr {
+	uint64_t	msg_id;
+	uint64_t        conn_id;
+	uint8_t		type;
+};
+
+struct rxm_op_hdr {
+	uint64_t	size;
+	uint64_t	data;
+	uint64_t	tag;
+	uint32_t	flags;
+	uint8_t		op;
+};
```

Comment:
The above structures aren't packed, so should both end up with padding.

Reply:
If this is a wire protocol change, we must update the version number and either handle operating between versions, or reject requests from peers using an unsupported version.  I would recommend moving anything that touches the protocol to a separate PR.

Reply:
> I would recommend moving anything that touches the protocol to a separate PR.
> 
> +1

Sure, will do that. Thanks